salt/client/ssh/__init__.py

```diff
@@ -147,7 +147,7 @@
 then SUDO="sudo "
 fi
 EX_PYTHON_INVALID={EX_THIN_PYTHON_INVALID}
-PYTHON_CMDS="python3 python27 python2.7 python26 python2.6 python2 python"
+PYTHON_CMDS="python3 /usr/libexec/platform-python3.6 python27 python2.7 python26 python2.6 python2 python"
```

Comment:
Just a minor question - why not doing it more generic? like:

```suggestion
PYTHON_CMDS="python3 /usr/libexec/platform-python python27 python2.7 python26 python2.6 python2 python"
```

Would that work?

Reply:
if that work, I would move it to the end of the list, so in case that another python package is installed, that will be used

Reply:
well, python 3 should be prefered over python 2.
The code iterrate over the list and the first wins.

Reply:
There's also a `/usr/libexec/platform-python` in the package which links to `/usr/libexec/platform-python3.6`. I can add this to the end of the list as well, but I think it's not really necessary.

Reply:
@meaksh I wanted to explicitly add the version so the ordering is clear, but since it's not `3` but `3.6`, I think your suggestion makes sense to future-proof this.